homeassistant/components/meraki/device_tracker.py

```diff
@@ -111,6 +112,8 @@ def _handle(self, hass, data):
                 gps_location = (lat, lng)
 
             attrs = {}
+            '''Device name only provided if the device has a name within merki dashboard otherwise setting name to Device Mac Address'''
```

Comment:
```suggestion
            # Device name only provided if the device has a name within meraki dashboard
            # otherwise setting name to Device Mac Address'''
```